From: Russ Allbery <eagle@eyrie.org>
Date: Sun, 24 Aug 2014 00:24:53 -0700
Subject: Increase buffer for network timeout tests

Newer versions of the Linux kernel use a larger internal buffer
for network writes, meaning that the test that tries to force a
timeout has to write more data.  Increase the data size to 8MB.
---
 tests/util/network/client-t.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/util/network/client-t.c b/tests/util/network/client-t.c
index 13bce09..aae31cd 100644
--- a/tests/util/network/client-t.c
+++ b/tests/util/network/client-t.c
@@ -5,7 +5,7 @@
  * which can be found at <http://www.eyrie.org/~eagle/software/rra-c-util/>.
  *
  * Written by Russ Allbery <eagle@eyrie.org>
- * Copyright 2005, 2013 Russ Allbery <eagle@eyrie.org>
+ * Copyright 2005, 2013, 2014 Russ Allbery <eagle@eyrie.org>
  * Copyright 2009, 2010, 2011, 2012, 2013
  *     The Board of Trustees of the Leland Stanford Junior University
  *
@@ -376,8 +376,8 @@ test_network_write(void)
     char *buffer;
 
     /* Create the data that we're going to send. */
-    buffer = bmalloc(4096 * 1024);
-    memset(buffer, 'a', 4096 * 1024);
+    buffer = bmalloc(8192 * 1024);
+    memset(buffer, 'a', 8192 * 1024);
 
     /* Create the listening socket. */
     fd = network_bind_ipv4(SOCK_STREAM, "127.0.0.1", 11119);
@@ -414,7 +414,7 @@ test_network_write(void)
      * A longer write cannot be completely absorbed before the client sleep,
      * so should fail with a timeout.
      */
-    ok(!network_write(c, buffer, 4096 * 1024, 1),
+    ok(!network_write(c, buffer, 8192 * 1024, 1),
        "network_write aborted with timeout");
     is_int(ETIMEDOUT, socket_errno, "...with correct error");
     alarm(0);
